Sources/Core/Sources/ECS/Singles/InputState.swift

```diff
@@ -86,11 +94,29 @@ public final class InputState: SingleComponent {
   /// Ticks the input state by flushing ``newlyPressed`` into ``keys`` and ``inputs``, and clearing
   /// ``newlyReleased``. Also emits events to the given ``EventBus``.
   func tick(_ isInputSuppressed: [Bool], _ eventBus: EventBus) {
+    /// increment the tick count
```

Comment:
Regular comments should use `//` instead of `///`. `///` is only used for documentation comments which `DocC` uses to generate documentation.